src/events/SDL_events.c

```diff
@@ -474,7 +474,7 @@ SDL_StopEventLoop(void)
         SDL_LockMutex(SDL_EventQ.lock);
     }
 
-    SDL_AtomicSet(&SDL_EventQ.active, 0);
+    SDL_EventQ.active = SDL_FALSE;
```

Comment:
Does this atomic change cause issues with caching and order of operations in multi-processor setups? As far as I understand most atomic APIs add guarantees for that sort of thing since they don't normally exist, even for boolean variables.